schemas/typescript/Event.ts

```diff
@@ -0,0 +1,17 @@
+// Generated by https://github.com/foxglove/schemas
+
+import { Duration } from "./Duration";
+import { EventKeyValue } from "./EventKeyValue";
+import { Time } from "./Time";
+
+/** a message used to annotate some event of interest in the data */
+export type Event = {
+  /** time when the event occurred */
+  timestamp: Time;
+
+  /** duration of event, starting at `timestamp` */
+  duration: Duration;
```

Comment:
Should duration be optional to indicate an instantaneous event? Or do we require specifying 0 length duration?

Reply:
optional imo (for schema formats that support optionals)